src/NHibernate/Cache/IBatchableCache.cs

```diff
@@ -1,41 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace NHibernate.Cache
-{
-	/// <summary>
-	/// Defines methods for retrieving and adding multiple objects from/to the cache at once.
-	/// The implementor should use this interface along with <see cref="ICache"/> when the
-	/// cache supports a multiple get and put operation.
-	/// </summary>
-	/// <remarks>
-	/// <threadsafety instance="true" />
-	/// <para>
-	/// All implementations <em>must</em> be threadsafe.
-	/// </para>
-	/// </remarks>
-	public partial interface IBatchableCache : IBatchableReadOnlyCache
```

Comment:
Directly added to `CacheBase` instead. (And we can drop this interface as it is not yet released.)